test/Dialect/Handshake/compose-buffers.mlir

```diff
@@ -24,3 +24,9 @@ handshake.func @foo(%arg0 : i32, %ctrl : none) -> (i32, none) {
   %2:2 = fork [2] %1 : i32
   return %2#1, %ctrl : i32, none
 }
+
+// -----
+
+// CHECK-LABEL:   handshake.func @external(
+// CHECK-SAME:      i32, none, ...) -> none
+handshake.func @external(%arg0: i32, %ctrl: none, ...) -> none
```

Comment:
Why is this here?

Reply:
Oh, I thought that this is the file to test buffer insertion. It seems that there is in fact none ;/
This case wasn't tested before, so while refactoring, I added a check for external functions and added a test.

I should have maybe split this into a separate PR...